mobile/apps/artwork/components/meta_data/templates/inquiry.jade

```diff
@@ -1,7 +1,8 @@
-unless artwork.auction 
-  if artwork.is_acquireable
-    a.artwork-meta-data-black__contact-button.js-purchase Buy
-  else if artwork.is_inquireable
+if artwork.is_acquireable
```

Comment:
I saw `is_purchasable` on the desktop side and was wondering if I should actually go with that? Looking [in Metaphysics](https://github.com/damassi/metaphysics/blob/master/src/schema/artwork/index.js#L362) it seems to be a bit more comprehensive, but not sure if I'm missing something subtle in the ask.

Reply:
I would keep the logic the same and use `is_acquireable`-- I think that `is_purchasable` is slightly different (refers to whether or not the artwork is enabled for "purchase request" which is separate from the e-commerce flow).

Reply:
Cool, so this is good to go 👍